app/controllers/application_controller.rb

```diff
@@ -11,4 +13,17 @@ def set_locale
   def default_url_options
     {locale: I18n.locale}
   end
+
+  protected
+  def get_layout
+    if logged_in?
+      if current_user.role == 1
+        "layouts/admin_layouts/application"
+      else
+        "layouts/user_layouts/application"
+      end
+    else
+      "layouts/user_layouts/application"
+    end
```

Comment:
```suggestion
      return  "layouts/user_layouts/application" unless logged_in? && current_user.admin?
      "layouts/admin_layouts/application"
```
Thử xem như này có hay hơn không em